bin/GameIndex.dbf

```diff
@@ -278,6 +278,7 @@ Region = NTSC-Unk
 Serial = SCAJ-20015
 Name   = Shin Megami Tensei III: Nocturne
 Region = NTSC-Unk
+eeRoundMode = 0 // Ladder glitch in "Assembly of Nihilo B11" level
```

Comment:
Period.

Reply:
Mate, this is just a comment in a db file. Not a statement in an essay or something,  this is too nitpicky. :/

Reply:
For the past few months we've been trying to keep the DB a bit cleaner and proper, let's keep it that way, besides there are already pending changes so why not do these as well while at it.

Reply:
I'm not sure how the lack of a period in a single statement is not clean for you, I'd agree with you in case it's a multiple line comment without punctuation but that's not the case here. Even with or without the period, the readability is not affected in any way.

If you really want to keep the GameDB clean, maybe notice the grammatical mistakes like ``This game need the EE cache `` in the commit which are more annoying than the lack of a period without significant meaning. Also there's no standard rule saying that a comment must have a full stop, let's not give extra work for the contributor based on personal opinion when they haven't done anything wrong.

And also I think we're fussing too much about keeping GameDB clean, I'd be concerned if it's a state like this,

```
Serial = SLES-50713
 Name   =              Freaky Flyers
         Region =         PAL-E
vuRoundMode = 1       // Fixes some bad effects.
```

rather than something like this which you changed in your recent GameDB clean up commit
```patch
-// allows import of Xenosaga I, Xenosaga I Reloaded, and Xenosaga Freaks data
+// Allows import of Xenosaga I, Xenosaga I Reloaded, and Xenosaga Freaks data.
```

Overall I feel like it's a really minor thing which doesn't need attention on, focusing on the functional change over these meaningless syntaxes would be much better in my opinion.

Reply:
Agreed, I don't think anyone without high levels of OCD cares if the comment in the gamedb file misses a period. Don't get me wrong I'm all for improving and cleaning it up, but as said above there are various other stuff that need fixing and are certainly much more important than some periods :P